PathOfTerraria.csproj

```diff
@@ -23,6 +23,12 @@
     </Compile>
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="NetEasy">
+      <HintPath>lib\NetEasy_v0.1.8.dll</HintPath>
```

Comment:
Not seeing these files committed - Are they ignored?

Reply:
Oh, I removed those since we're not using NetEasy anymore. I'll fix that